### PR TITLE
add documentation for Querying Metrics from Prometheus

### PR DESCRIPTION
**Pull request by LiZhenCheng9527**
add documentation for Querying Metrics from Prometheus

### CODE REVIEW

File: content/en/docs/userguide/metric.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+Metrics monitored by Kmesh L4 at this stage:
+
+|Name|Describe|
+|---|---|
+|kmesh_tcp_connections_opened_total|The total number of TCP connections opened|
+|kmesh_tcp_connections_closed_total|The total number of TCP connections closed|
+|kmesh_tcp_received_bytes_total|The size of total bytes received during request in case of a TCP connection|
+|kmesh_tcp_sent_bytes_total|The size of total bytes sent during response in case of a TCP connection|
+
+Here's how to view these metrics throught Prometheus:
```
Comment (YaoZengzeng) [inline content/en/docs/userguide/metric.md:71]:
```suggestion
Here's how to view these metrics through Prometheus:
```

File: content/en/docs/userguide/metric.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+**1. Verify that the prometheus service is running in your cluster.**
+
+In Kubernetes environments, execute the following command:
```
Comment (YaoZengzeng) [inline content/en/docs/userguide/metric.md:75]:
```suggestion
In Kubernetes environment, execute the following command:
```

File: content/en/docs/userguide/metric.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+Please refer [quickstart](https://kmesh.net/en/docs/setup/quickstart/)
+
+Note that the following should be added to the Kmesh annotation:
+
+```console
+annotations:
+    prometheus.io/path: "status/metric"
+    prometheus.io/port: "15020"
+    prometheus.io/scrape: "true"
```
Comment (YaoZengzeng) [inline content/en/docs/userguide/metric.md:26]: Haven't these annotations been added to the Kmesh DaemonSet via https://github.com/kmesh-net/kmesh/pull/589 ?

File: content/en/docs/userguide/metric.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+In Kubernetes environments, execute the following command:
+
+```console
+kubectl -n kmesh-system get svc prometheus
+
+NAME         TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)    AGE
+prometheus   ClusterIP   10.96.18.252   <none>        9090/TCP   24h
+```
+
+**2. Send traffic to the mesh.**
```
Comment (YaoZengzeng) [inline content/en/docs/userguide/metric.md:84]: It's weired, not send traffic to the mesh, but the traffic between applications in the mesh.

File: content/en/docs/userguide/metric.md [hunk 5]
```diff
@@ -0,0 +1,135 @@
+---
+draft: false
+linktitle: Querying Metrics from Prometheus
```
Comment (YaoZengzeng) [inline content/en/docs/userguide/metric.md:3]:
```suggestion
linktitle: Querying L4 Metrics from Prometheus
```

As I will add a doc about showing L7 metrics :)

File: content/en/docs/userguide/metric.md [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+Note that the following should be added to the Kmesh annotation:
+
+```console
+annotations:
+    prometheus.io/path: "status/metric"
+    prometheus.io/port: "15020"
+    prometheus.io/scrape: "true"
+```
+
+2. Using Kmesh manage default namespace
```
Comment (hzxuzhonghu) [inline content/en/docs/userguide/metric.md:29]:
```suggestion
2. Using Kmesh to manage `default` namespace
```

File: content/en/docs/userguide/metric.md [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+
+kubectl apply -f https://raw.githubusercontent.com/kmesh-net/kmesh/main/samples/sleep/sleep.yaml
+
+kubectl get po -A 
+NAMESPACE            NAME                                          READY   STATUS    RESTARTS   AGE
+default              sleep-bc9998558-pbfvk                         1/1     Running   0          7m
+default              tcp-echo-7f676db574-mzmql                     1/1     Running   0          7m
+```
+
+Note: Confirm that sleep and tcp-echo are indeed [managed by kmesh](https://kmesh.net/en/docs/setup/quickstart/#deploy-the-sample-applications).
```
Comment (hzxuzhonghu) [inline content/en/docs/userguide/metric.md:58]: Bold

File: content/en/docs/userguide/metric.md (hunk 1 above)
Comment (hzxuzhonghu) [reply content/en/docs/userguide/metric.md:71]: seems we need to introduce codespell github action to this repo too @Okabe-Rintarou-0

File: content/en/docs/userguide/metric.md [hunk 8]
```diff
@@ -0,0 +1,135 @@
...
+In the “Expression” input box at the top of the web page, enter the text:
+
+```console
+kmesh_tcp_connections_opened_total
+```
+
+The results will be similar to:
+
+<div align="center">
+<img src="/docs/userguide/metric/prometheus-table.png" width="1400" />
```
Comment (hzxuzhonghu) [inline content/en/docs/userguide/metric.md:116]: The image is too small, icannot see the words even zoom out

File: content/en/docs/userguide/metric.md (hunk 3 above)
Comment (LiZhenCheng9527) [reply content/en/docs/userguide/metric.md:26]: Emphasise that these annotations

**Review by hzxuzhonghu — APPROVED** [review]
/lgtm

File: content/en/docs/userguide/metric.md [hunk 9]
```diff
@@ -0,0 +1,148 @@
...
+
+<div align="center">
+<img src="/docs/userguide/metric/prometheus-graph.png" width="1400" />
+</div>
+
+### Cleanup
+
+Disable port-forward.
+
+cleanup Prometheus:
```
Comment (hzxuzhonghu) [inline content/en/docs/userguide/metric.md:142]: use numbered list

Comment (hzxuzhonghu) [inline content/en/docs/userguide/metric.md:142]: Cleanup

File: content/en/docs/userguide/metric.md [hunk 10]
```diff
@@ -0,0 +1,148 @@
...
+|kmesh_tcp_workload_connections_closed_total|The total number of TCP connections closed to a workload|
+|kmesh_tcp_workload_received_bytes_total|The size of the total number of bytes received in response to a workload over a TCP connection|
+|kmesh_tcp_workload_sent_bytes_total|The size of the total number of bytes sent in response to a workload over a TCP connection|
+|kmesh_tcp_workload_conntections_failed_total|The total number of TCP connections failed to a workload|
+
+**Service:**
+
+|Name|Describe|
+|---|---|
+|kmesh_tcp_connections_opened_total|The total number of TCP connections opened to a service|
```
Comment (hzxuzhonghu) [inline content/en/docs/userguide/metric.md:78]: is this still valid, vagulaly remember is it with _service_

Comment (LiZhenCheng9527) [reply content/en/docs/userguide/metric.md:78]: metric name changed.

https://github.com/kmesh-net/kmesh/blob/155a8a5dcf542663b7108129dace7729677370d8/pkg/controller/telemetry/utils.go#L148-L177

**Pull request merged**